### PR TITLE
JESI-1743

### PR DESCRIPTION
**Pull request by AndreTheHunter**
[JESI-1743](https://jesims.atlassian.net/browse/JESI-1743)
- [x] https://github.com/jesims/parent/pull/46
- [x] https://github.com/jesims/bindle/pull/13
- [x] https://github.com/jesims/backpack/pull/66
- [x] Update changelog
- [x] https://github.com/jesims/bindle/pull/15
- [x] https://github.com/jesims/backpack/pull/69 and remove `-SNAPSHOT`

### CODE REVIEW

File: src/io/jesi/customs/strict.cljc [hunk 1]
```diff
@@ -40,41 +40,42 @@
   matches (with `re-find`) the regular expression `re`."
   ([form]
    {:pre [(some? form)]}
-   `(env/transform
-      (test/is ~form)))
+   (let [is (env/symbol &env `test/is)]
```
Comment (AndreTheHunter) [inline src/io/jesi/customs/strict.cljc:43]: `StackOverflowError` still occurs even when not using `env/transform`, but using `env/symbol` should lead to faster compile times at least. The workaround is to create a cljs specific version of strict (not sure why is works). See `jesi-cljs/test/io/jesi/cljs/test/strict.cljc`

File: project.clj [hunk 2]
```diff
@@ -7,20 +7,19 @@
...
-  :clean-targets [:target-path :compile-path "out"]
-  :parent-project {:coords  [io.jesi/parent "3.3.0"]
+  :plugins [[lein-parent "0.3.8"]]
+  :clean-targets ^{:protect false} [".shadow-cljs" "out" :target-path]
+  :parent-project {:coords  [io.jesi/parent "3.9.0"]
                    :inherit [:plugins :managed-dependencies :deploy-repositories :dependencies :profiles :test-refresh :aliases :codox]}
   :managed-dependencies [[com.google.guava/guava "23.0"]
-                         [io.jesi/backpack]]
-  :dependencies [[io.jesi/backpack "4.2.1"]                 ;TODO use managed version (at least 5.0)
+                         [io.jesi/backpack "5.1.2-SNAPSHOT"]] ;FIXME use managed 5.1.2 version
```
Comment (axrs) [inline project.clj]: reminder

File: project.clj [hunk 3]
```diff
@@ -7,20 +7,19 @@
...
+  :dependencies [[io.jesi/backpack]
                  ;CLJ
                  [org.clojure/clojure :scope "provided"]
                  [leiningen "2.9.1" :exclusions [org.slf4j/slf4j-nop]]
                  [org.clojure/tools.namespace "0.3.1"]
                  ;CLJS
-                 [org.clojure/clojurescript :scope "provided"]
-                 [thheller/shadow-cljs :scope "provided"]
+                 [thheller/shadow-cljs]
```
Comment (axrs) [inline project.clj]: Should this be a dependency? Or a dev dependency?

File: test/io/jesi/customs/spy_test.cljc [hunk 4]
```diff
@@ -145,11 +153,13 @@
...
+          (is= (str file (set-line 1) " a:" \newline "1" \newline)
                (with-out-str (reset! result (spy/ppeek a))))
           (is= a @result)
 
           (testing "even in a thread macro (no line numbers since the &from metadata is not preserved)"
-            (is= (str file " a:" \newline "1" \newline)
-                 (with-out-str (reset! result (-> a spy/ppeek inc))))
-            (is= (inc a) @result)))))))
+            (let [file (if (cljs?) ns-name file)]
+              (println "file:" file)
```
Comment (axrs) [inline test/io/jesi/customs/spy_test.cljc]:
```suggestion
```

**Review by axrs — CHANGES_REQUESTED** [review]

File: project.clj (hunk 3 above)
Comment (AndreTheHunter) [reply project.clj]: Done.
Yup, dev dependency and `clojurescript` as provided

**Review by axrs — APPROVED** [review]

**Pull request merged**